pkg/proto/protobuf_converters.go

```diff
@@ -351,7 +351,6 @@ func (c *ProtobufConverter) attachment(att *g.Attachment, untyped bool) Attachme
 		return nil
 	}
 	if att == nil {
-		c.err = errors.New("empty attachment")
```

Comment:
In `pkg/ride/evaluator/ast/transactions.go` we use tx.Attachment. We should modify code to check Attachments for nil there.